rules/autoconsent/microcenter.com.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "microcenter.com",
+    "vendorUrl": "https://www.microcenter.com/",
```

Comment:
It’s an optional field so shouldn’t we omit `vendorUrl` for site specific rules?